tests/emukit/quadrature/test_measures.py

```diff
@@ -4,10 +4,15 @@
 
 import numpy as np
 import pytest
+from math import isclose
```

Comment:
numpy has many utilities to make life easier: isclose, allclose, assert_allclose, assert_almost_equal, maybe even more. Can any of them help you here?

Reply:
I like to use math.isclose because of this https://github.com/numpy/numpy/issues/10161

They even state a waring here https://docs.scipy.org/doc/numpy/reference/generated/numpy.isclose.html

what are your thoughts?

Reply:
So i read the thread, at least some of it. thanks for sharing, had no clue!

numpy.isclose is indeed flawed, but assert_allclose isn't. can we use this one?

Reply:
do you mean this? https://docs.scipy.org/doc/numpy-1.14.1/reference/generated/numpy.testing.assert_allclose.html

I think it is flawed, too because it relies on this which is basically elementwise isclose 
https://docs.scipy.org/doc/numpy/reference/generated/numpy.allclose.html

We often deal with small values, especially when we compute gradients, so I kind of don't like how numpy deals with it.

Reply:
ok, i obviously need to dive deeper to understand this issue. for now consider the comment retracted